EXILED/Exiled.API/Features/Player.cs

```diff
@@ -687,24 +687,24 @@ public ScpSpawnPreferences.SpawnPreferences ScpPreferences
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
         // TODO: Change logic for FacilityGuard in next major update
-        public bool IsNTF => Role?.Team is Team.FoundationForces;
+        public bool IsNTF => Role?.Type.IsNtf() ?? false;
```

Comment:
Keep

Reply:
Should I just revert that whole file then or is there something I should keep because I am getting mixed everything over here. I don't know what is wanted and what is not wanted. Please someone just give me a straight answer.